_docs/gov-dev/fast-track.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Fast-track an External Proposal
+subtitle: Learn how to fast-track an external proposal with Technical Committee via the polkadot.js app
+author: charles
+tags: [governance]
+---
+# Fast track
```

Comment:
@imstar15 I broke the one file into 3 because they were for different topics, however, they lost context after the break-down. 

Could you add some context back to this fast-track page? For example, external proposals need to be created as a pre-requisite.

Also, briefly explain some other operations that a tech committee can do. For example, `vetoExternal` and vote against an external proposal, and cancel Referendum can cancel an on-going referendum.